search-app/src/cmr/search/services/query_service.clj

```diff
@@ -25,6 +27,7 @@
    [cmr.common.mime-types :as mt]
    [cmr.common.services.errors :as errors]
    [cmr.common.util :as u]
+   [cmr.indexer.data.concepts.granule :as granule]
```

Comment:
Search should not be using indexer namespaces (and doesn't have indexer app in its library dependencies so this would fail when running outside of dev-system).